tests/SwiftBuildTool/Inputs/swiftc-shim.cpp

```diff
@@ -0,0 +1,17 @@
+#include <stdlib.h>
+#include <string>
+int main(int argc, char** argv) {
+  std::string args;
+  for (int i = 1; i < argc; i++) {
+    args += " \"";
+    args += argv[i];
```

Comment:
This doesn't escape arguments properly, but it seems overkill to link against llvm or implement the logic for the handful of tests that use this.

Reply:
Thats fine with me.

I would call this file `pseudo-swiftc-shim`, for consistency.